controller/src/main/java/io/pravega/controller/server/eventProcessor/ControllerEventProcessors.java

```diff
@@ -121,6 +122,8 @@
     private final long rebalanceIntervalMillis;
     private final AtomicLong truncationInterval;
     private ScheduledExecutorService rebalanceExecutor;
+    @Getter
+    private boolean isReady = false;
```

Comment:
We need a zookeeper connectivity check for ControllerEventProcessor as well, as this too depends on Zookeeper..... perhaps I missed mentioning this on the issue.... If `CheckpointStore` is of type `ZKCheckpointStore` we need to check connectivity to Zookeeper.

Reply:
Added that

Reply:
cool. thanks!